src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[],
+  filter: (filter: Filter) => void
+  removeCompletedTodos: () => void,
+  currentFilter: Filter
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  removeCompletedTodos: removeAll,
```

Comment:
Consider renaming `removeCompletedTodos` prop to `removeAll` for better clarity and consistency. 🤔

Reply:
or rename `removeAll` to `removeCompleted`